docs/spring-boot-runtime/master.adoc

```diff
@@ -244,15 +244,13 @@ The recommended approach for specifying and using supported and tested maven art
 
 // workaround to resolve ID conflicts:
 :context: spring-boot
+//creating an app from scratch
+include::topics/assembly_creating-a-basic-spring-boot-application.adoc[leveloffset=+2]
```

Comment:
I have created separate assemblies for the individual runtimes to facilitate the reuse of modules (it was getting very messy).